src/utils/jwt.ts

```diff
@@ -118,6 +118,48 @@ export class JWT {
 
     return jwt.sign(opts.params, keySecret, tokenOptions);
   }
+  /**
+   * Creates a new token to be used with a signed Space ID
+   * @param {string} spaceId - The Space ID (of type 'signed') that you'd like to generate a token for.
+   * @param {Object} options - Configuration options to use when creating the token
+   * @param {string} [options.keyId] - The signing key ID to use. If not specified, process.env.MUX_SIGNING_KEY is attempted
+   * @param {string} [options.keySecret] - The signing key secret. If not specified, process.env.MUX_PRIVATE_KEY is used.
+   * @param {string} [options.type=rt] - Type of token this will be. Which is rt for Real-time
+   * @param {string} [options.expiration=7d] - Length of time for the token to be valid.
+   * @param {Object} [options.params] - Any additional query params you'd use with a public url.
+   * @returns {string} - Returns a token to be used with a signed URL.
+   *
+   * @example
+   * const Mux = require('@mux/mux-node');
+   *
+   * const token = Mux.JWT.signSpaceId('some-space-id', { keyId: 'your key id', keySecret: 'your key secret' });
+   */
+  static signSpaceId(spaceId: string, options: MuxJWTSignOptions = {}) {
+    const opts = {
+      type: 'rt',
```

Comment:
I don't think we should allow overriding of `type` for the `signSpaceId` prototype, it should be fixed to `rt`.

Reply:
Yea that's fair